lib/hub/commands.rb

```diff
@@ -347,12 +347,6 @@ def land(args)
       end
     end
 
-    # $ hub e-note
-    # $ hub e-note "My humble contribution"
-    # $ hub e-note -i 92
-    # $ hub e-note https://github.com/rtomayko/tilt/issues/92
-    alias_method :e_note, :pull_request
```

Comment:
haha thats amazing - thanks for sharing

On Tue, Feb 23, 2016 at 10:27 AM, André Arko <notifications@github.com>
wrote:

> In lib/hub/commands.rb
> <https://github.com/Gusto/hub/pull/5#discussion_r53824749>:
>
> > @@ -347,12 +347,6 @@ def land(args)
> >        end
> >      end
> >
> > -    # $ hub e-note
> > -    # $ hub e-note "My humble contribution"
> > -    # $ hub e-note -i 92
> > -    # $ hub e-note https://github.com/rtomayko/tilt/issues/92
> > -    alias_method :e_note, :pull_request
>
> it was an employee joke after Fox News tried to explain Pull Requests as
> "e-notes": https://twitter.com/huth/status/378306063520374784
>
> since no one actually uses it, it didn't seem worth maintaining in our
> fork :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Gusto/hub/pull/5/files#r53824749>.
>